.github/workflows/versionBumpCheck.yml

```diff
@@ -0,0 +1,29 @@
+name: Version Bump Check
+# Description:
+  # In this workflow we check for keywords in commit message to determine if
+  # a specific version bump is necessary. This check runs on every push and can
+  # perform version updates before and during a pull request.
+
+on: push
+
+jobs:
+  check_push_version_bump:
+    name: 'Check Push Version Bump'
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Checkout source code"
+        uses: actions/checkout@v2
+      - name: "Setup Node.js"
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: "Automated Version Bump (Push)"
+        uses: 'phips28/gh-action-bump-version@master'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          skip-tag:  'true'
+          minor-wording:  'minor,Minor,MINOR,add,Adds,ADD,ADDS,new,New,NEW,feat,Feat,FEAT,feature,Feature,FEATURE,features,Features,FEATURES'
```

Comment:
Thanks. So this tells me that the keyword check is case-sensitive.

Reply:
Yes with the current script. We can pretty easily fork the gh-action-bump-version and edit it to be case insensitive. I figured we can save that for a later implementation.